biz.aQute.bndlib/src/aQute/bnd/osgi/Jar.java

```diff
@@ -486,25 +669,34 @@ Optional<ModuleAttribute> moduleAttribute() throws Exception {
 				module_info = ClassFile.parseClassFile(din);
 			}
 		}
-		return moduleAttribute = Arrays.stream(module_info.attributes)
+		return Arrays.stream(module_info.attributes)
 			.filter(ModuleAttribute.class::isInstance)
 			.map(ModuleAttribute.class::cast)
 			.findFirst();
 	}
 
 	public String getModuleName() throws Exception {
-		return moduleAttribute().map(a -> a.module_name)
-			.orElseGet(this::automaticModuleName);
+		return getModuleName(MULTI_RELEASE_DEFAULT_VERSION);
 	}
 
-	String automaticModuleName() {
```

Comment:
It looks like we lost a method here, which would break version compatibility.It could just delegate to the new method with version `0`.

Reply:
the method is package private an nowhere used in the package, but if you think it helps, I can add it again...

Reply:
I know that removing methods is likely to trigger a baselining error (though default visibility may be safe), and if we see a baselining error it will incur the wrath of @bjhargrave :smile:

Reply:
Maybe you can enable the workflow runs so we can get a normative result while I address the mentioned issues? :-)

Reply:
Done.